modules/ossm-threescale-istio-adapter-troubleshooting.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v1x/threescale_adapter/threescale-adapter.adoc
+// * service_mesh/v2x/threescale_adapter/threescale-adapter.adoc
+
+[id="ossm-threescale-istio-adapter-verification-troubleshooting_{context}"]
+= 3scale Istio adapter verification and troubleshooting
+
+You might want to check whether the 3scale Istio adapter is working as expected. If your adapter is not working, use the following steps to help troubleshoot the problem.
+
+.Procedure
+
+. Ensure the _3scale-adapter_ pod is running in the control plane namespace:
++
+----
+$ oc get pods -n <istio-system>
```

Comment:
Formatting needs to match OpenShift standards, please add [source,terminal] above both of your commands, as shown below:

```
[source,terminal]
----
$  oc get pods -n <istio-system>
----
```

Reply:
Thanks for catching this.